lib/ts/nextjs/ssr.ts

```diff
@@ -135,14 +125,13 @@ export default class SuperTokensNextjsSSRAPIWrapper {
     }
 
     /**
-     * Authenticates a server action and then passes the session context as a parameter
+     * Ensures that a server action is called by an authenticated user
      * If the session does not exist/user is not authenticated, it will automatically redirect to the login page
      * The function is meant to run on the client, before calling the actual server action
-     * @param action - A server action that takes the session context as its first parameter
+     * @param action - A server action that will get called after the authentication state is confirmed
      * @returns The server action return value
      **/
-    static async authenticateServerAction<T extends (session: LoadedSessionContext) => Promise<K>, K>(action: T) {
-        let loadedSessionContext: LoadedSessionContext | undefined = undefined;
+    static async confirmAuthenticationAndCallServerAction<T extends () => Promise<K>, K>(action: T) {
```

Comment:
I think a shorter name would be better. 
```suggestion
    static async ensureSessionAndCall<T extends () => Promise<K>, K>(action: T) {
```